Missing dependency on Mint build instructions

I'm using Linux Mint 18.3 (based off Ubuntu 16.04). Running the build steps for the Ubuntu platform gives me the following errors for the `make` command:

```
llvm-config: error: component libraries and shared library

llvm-config: error: missing: /usr/lib/llvm-3.9/lib/libLLVMSupport.a
llvm-config: error: missing: /usr/lib/llvm-3.9/lib/libLLVMCore.a
llvm-config: error: missing: /usr/lib/llvm-3.9/lib/libLLVMProfileData.a
... snip ...
llvm-config: error: missing: /usr/lib/llvm-3.9/lib/libLLVMObjCARCOpts.a
llvm-config: error: missing: /usr/lib/llvm-3.9/lib/libLLVMLTO.a
genreference.c
In file included from src/libponyc/codegen/genreference.h:5:0,
                 from src/libponyc/codegen/genreference.c:1:
src/libponyc/codegen/codegen.h:5:25: fatal error: llvm-c/Core.h: No such file or directory
compilation terminated.
Makefile:847: recipe for target 'build/release/obj-native/libponyc/codegen/genreference.o' failed
make: *** [build/release/obj-native/libponyc/codegen/genreference.o] Error 1
```

This was fixed by installing the `llvm-3.9-dev` dependency with the archived codes.

I haven't tested the build process yet on the default Ubuntu installation, or on Linux Mint 19 / Ubuntu 18.04.

--- a/README.md
+++ b/README.md
@@ -162,6 +162,7 @@ wget https://copr-be.cloud.fedoraproject.org/results/ponylang/ponylang/pubkey.gp
 rpm --import pubkey.gpg
 zypper install ponyc
 ```
+
 ### Using Zypper for OpenSuSE Tumbleweed:
 
 ```bash
@@ -170,7 +171,6 @@ wget https://copr-be.cloud.fedoraproject.org/results/ponylang/ponylang/pubkey.gp
 rpm --import pubkey.gpg
 zypper install ponyc
 ```
-
 
 ### RPM and AVX2 Support
 
@@ -203,6 +203,31 @@ sudo apt-get -V install ponyc
 ### DEB and AVX2 Support
 
 By default, the Pony DEB package is compiled without support for AVX CPU instructions. For optimal performance, you should build your Pony installation from source.
+
+### Linux Mint
+
+All steps to install Pony in Linux Mint are the same from Ubuntu, but you must use the Ubuntu package base (`trusty`, `xenial`, `bionic`) instead of the Linux Mint release.
+
+Install pre-requisites and add the correct `apt` repository:
+
+```bash
+sudo apt-get install \
+     apt-transport-https \
+     ca-certificates \
+     curl \
+     gnupg2 \
+     software-properties-common
+```
+
+```bash
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "E04F0923 B3B48BDA"
+source /etc/upstream-release/lsb-release
+sudo add-apt-repository "deb https://dl.bintray.com/pony-language/ponylang-debian $DISTRIB_CODENAME main"
+sudo apt-get update
+sudo apt-get -V install ponyc
+```
+
+The same AVX2 support restrictions apply.
 
 ## Gentoo Linux
 
@@ -446,7 +471,7 @@ make
 ./build/release/ponyc examples/helloworld
 ./helloworld
 ```
-### Ubuntu Artful
+### Ubuntu Bionic and later
 
 ```bash
 sudo apt-get update
@@ -467,6 +492,16 @@ cd ~/ponyc/
 make default_pic=true
 ./build/release/ponyc examples/helloworld
 ./helloworld
+```
+
+### Linux Mint
+
+Instructions for Linux Mint are the same as the appropriate Ubuntu installation. However, an extra `llvm-3.9-dev` package is required for missing headers.
+
+After installing the `llvm` package by following the appropriate steps for Ubuntu Trusty (Linux Mint 17), Xenial (Linux Mint 18), or Bionic (Linux Mint 19), install the extra headers:
+
+```bash
+sudo apt-get install -y llvm-3.9-dev
 ```
 
 ### Fedora (25)

--- a/README.md
+++ b/README.md
@@ -471,7 +471,7 @@ make
 ./build/release/ponyc examples/helloworld
 ./helloworld
 ```
-### Ubuntu Bionic and later
+### Ubuntu Bionic
 
 ```bash
 sudo apt-get update
@@ -489,7 +489,7 @@ Build ponyc, compile and run helloworld:
 
 ```bash
 cd ~/ponyc/
-make default_pic=true
+make default_pic=true default_ssl=openssl_1.1.0
 ./build/release/ponyc examples/helloworld
 ./helloworld
 ```

